packages/shared/src/components/cards/SharePostCard.tsx

```diff
@@ -93,7 +88,7 @@ export const SharePostCard = forwardRef(function SharePostCard(
             <CardSpace />
             <PostTags tags={post.sharedPost.tags} />
             <PostMetadata
-              createdAt={post.sharedPost.createdAt}
+              createdAt={postCreatedAt}
```

Comment:
Looking at this, it could just be `post.createdAt`, as this entire block is inside a `improvedSharedPostCard` ternary on line 73.
So we don't need `postCreatedAt` constant.